<h1>
 Title: AssertionError
</h1>
<h3>
 Author: Anonymous
</h3>
<h3>
 Date: 2021-05-15T08:21:52.351447+10:00
</h3>
<h3>
 Category: Assignments
</h3>
<h3>
 Vote Count: 0
</h3>
<p>
 Traceback (most recent call last):
</p>
<p>
 File "schedule_tests.py", line 299, in &lt;module&gt;
</p>
<p>
 testfn() # Adding the parentheses calls whatever testfn is
</p>
<p>
 File "schedule_tests.py", line 102, in test_studentschedule_add_noconflicts
</p>
<p>
 assert set(c.in_schedules) == set([my_schedule, roster]), \
</p>
<p>
 AssertionError: AEP 3560 LEC 001.in_schedules should be ['Cornell Fall 21 roster', 'My Plan A'], but is ['Cornell Fall 21 roster']
</p>
<p>
 This error appeared after I edited something in my conflictsWith() function (I hadn't touched any part of my StudentSchedule or Course code).
</p>
<h3>
 ----------- REPLIES -----------
</h3>